src/Soil-Core-Tests/SoilPersistentDictionaryTest.class.st

```diff
@@ -66,10 +64,7 @@ SoilPersistentDictionaryTest >> testCommitAndRead [
 
 	transaction := soil newTransaction.
 	materialized := transaction root.
-
-	"should this not be transparently load it?"
-	"if the Proxy inherits from ProtoObject, it works"
-	self assert: (materialized at: #test) class equals: SOObjectProxy
+	self assert: (materialized at: #test) equals: 'string'
```

Comment:
I don't know why the objects are being resolved but it is wrong. This defeats the purpose of having proxies